packages/firestore/test/unit/specs/spec_builder.ts

```diff
@@ -769,18 +770,19 @@ export class SpecBuilder {
     return this;
   }
 
-  watchFilters(queries: Query[], ...docs: DocumentKey[]): this {
```

Comment:
Original watchFilters accept an array of targetIds (limited to only 1 targetId), and spread operator to pass in unlimited number of docs(only to get count later on).
updated the watchFilters to accept array of targetIds, array of docs(default to empty array), and an optional bloom filter.